### ISSUE TITLE
Add keybindings for Tree Find Controls

### ISSUE THREAD

**Issue by bew** [microsoft/vscode#204045]
Since vscode 1.70 we have generic tree find controls, to search/filter any tree view with a pattern.

(As a user of `vscode-neovim`) I'm looking for ways to bind keys to the action of toggling strict filter (hide non-matches) and toggle fuzzy matching.
And when not using strict filtering, have actions to jump to the next/previous match in the tree.

![image](https://github.com/microsoft/vscode/assets/9730330/e00b460e-cad9-41e6-8a85-83a2642dcc3e)

**Comment by gjsjohnmurray** [comment]
I think #175137 just added keybindable `list.toggleFindMatchType`

**Comment by bew** [comment]
Thanks, it seems those new actions aren't available in a stable release yet, right?
(tried looking at for them in the latest January release from a few days)

**Comment by gjsjohnmurray** [comment]
Here's what I see:

![image](https://github.com/microsoft/vscode/assets/6726799/5c1707ff-3097-44d6-a901-d26780aee293)

**Comment by bew** [comment]
Ah yes indeed, I was looking for the action description like `explorer fuzzy` to match `Explorer: Switch to Fuzzy Find Match Type`.
Both actions work 👍

Is there a context expression to determine if the find widget is opened ? (didn't find `find*` variables in that context)

**Comment by gjsjohnmurray** [comment]
Do you know about https://code.visualstudio.com/api/references/when-clause-contexts#inspect-context-keys-utility?

Maybe `listSupportsFind && treeFindOpen` (untested) though this probably won't confine the keybinding to apply only when focus is in the find widget.

**Comment by bew** [comment]
> Do you know about https://code.visualstudio.com/api/references/when-clause-contexts#inspect-context-keys-utility?

Yes, that's what I used to determine there was no `find*` context vars, didn't think to hunt for `*find*` vars though 👍

Okay so my first request seems to be already fixed.

What do you think about the second about next/previous keys to jump from match to match?
